lib/database/repositories/LogRepository.js

```diff
@@ -37,6 +37,17 @@ class LogRepository extends ILogRepository {
         return Log.findAll().map(LogAdapter.toEntity);
     }
 
+    /**
+     * Returns a specific entity.
+     *
+     * @param {Number} id ID primary key of the entity to find.
+     * @returns {Promise|Null} Promise object representing the full mock data
+     */
+    async find(id) {
+        const result = await Log.findByPk(id);
+        return result ? [LogAdapter.toEntity(result)] : null;
```

Comment:
Actually I did this because of the openapi specification which already said that data must be an array. Can I change that to support arrays as well as objects?

Reply:
There is an `oneOf` option in the OpenAPI spec so perhaps have a look there.